SocialFilter/sections/specific.txt

```diff
@@ -2004,7 +2004,8 @@ indosport.com##div[title^="Bagikan Artikel Ini"]
 otvet.mail.ru##a[title="Поделиться"]
 houzz.ru##.hz-editorial-gallery-action-bar__share
 blog.esuteru.com##.comment-v2 + aside.section-box
-blog.esuteru.com##.plugin-twitter_follow_button
+vipsister23.com,blog.esuteru.com##.plugin-twitter_follow_button
```

Comment:
This also is empty even if AG is turned off, tho I think it's acceptable having the rule.

Reply:
I see